packages/sitecore-jss-vue/src/components/RichText.test.ts

```diff
@@ -111,4 +112,85 @@ describe('<RichText />', () => {
     expect(rendered.attributes()).toMatchObject(attrs);
     expect(rendered.element.innerHTML).toBe(props.field.value);
   });
+
+  it('should initialize links', () => {
```

Comment:
This test doesn't check that link is initialized, you just check markup
in order to check that link is initialized, you should try to click, you need to understand that `this.routeHandler` is assigned to the event listener, you have such test at the bottom